src/elements/content-sidebar/DocgenSidebar.js

```diff
@@ -0,0 +1,86 @@
+// @ts-check
+/**
+ * @flow
+ * @file Sign sidebar component
```

Comment:
should it be?
```suggestion
 * @file Doc Gen sidebar component
```

we might even change to use the full name. I'm not familiar with what this is though 
 document generation / generator?